src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -275,15 +285,22 @@ export const WalletOptions: Definitions<WalletConfig> = {
     cliAliases: ["u", "unlock"],
     cliType: "array:string"
   },
-  secure: {
+  lock: {
     normalize,
     cliDescription:
       "Lock available accounts by default (good for third party transaction signing).",
     default: () => false,
-    legacyName: "secure",
-    cliAliases: ["n", "secure"],
+    legacyName: "lock",
+    cliAliases: ["n", "secure", "lock"],
     cliType: "boolean"
   },
+  passphrase: {
+    normalize,
+    cliDescription: "Passphrase to use when locking accounts.",
+    default: () => "",
+    cliAliases: ["passphrase", "password"],
```

Comment:
I not convinced we need to alias this to `password`.

`cliAliases` create aliases at the top level (`ganache --passphrase`) instead of namespace (`ganache --wallet.passphrase`). Do you think we should add passphrase to the top level?


```suggestion
    cliAliases: ["passphrase"],
```